src/Datagrid/ListMapper.php

```diff
@@ -92,6 +92,15 @@ public function add($name, $type = null, array $fieldDescriptionOptions = [])
             $fieldDescriptionOptions['virtual_field'] = true;
         }
 
+        if (\array_key_exists('identifier', $fieldDescriptionOptions) && !\is_bool($fieldDescriptionOptions['identifier'])) {
+            @trigger_error(
+                'Passing a non boolean value for "identifier" option is deprecated since 3.x and will throw an exception in 4.0.',
+                E_USER_DEPRECATED
+            );
+            // NEXT_MAJOR: throw an exception instead
```

Comment:
Please add the exception commented so we only need to I comment in the next major.
Same for the test case, mark it as skipped? Or comment it out?

Reply:
Done!